qiskit/transpiler/preset_passmanagers/level3.py

```diff
@@ -201,6 +202,7 @@ def _opt_control(property_set):
             plugin_config=unitary_synthesis_plugin_config,
             target=target,
         ),
+        HighLevelSynthesis(hls_config=hls_config),
```

Comment:
```suggestion
```
I think for now we should leave this out of the optimization loop until we have collection passes that will yield high level objects it can synthesize.

Reply:
Yes, this makes sense, I have removed this line for now.